Source/Bogus/Faker[T].cs

```diff
@@ -740,14 +863,14 @@ private ValidationResult ValidateInternal(string[] ruleSets)
             //What's left in userSet is the set of properties or fields
             //that the user does not know about + .Rule() methods.
 
-            if( userSet.Count > 0 )
+            if (userSet.Count > 0)
             {
-               foreach( var propOrFieldOfT in userSet )
+               foreach (var propOrFieldOfT in userSet)
                {
-                  if( populateActions.TryGetValue(propOrFieldOfT, out var populateAction) )
+                  if (populateActions != null && populateActions.TryGetValue(propOrFieldOfT, out var populateAction))
```

Comment:
Mentioned bug was here when no RuleFor() was called (Or simple AddRule) when StrictMode(true) the populateActions  variable was null and resolved in null refernce exception